api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -350,7 +350,7 @@ message RequirementRule {
 //              - provider_name: "provider1"
 //              - provider_name: "provider2"
 //
-//// [#not-implemented-hide:]
+// [#not-implemented-hide:]
```

Comment:
Is this now implemented so we can remove this line? Also, can you add a line to `version_history.rst`?